core/src/main/resources/lib/form/password.jelly

```diff
@@ -60,74 +60,88 @@ THE SOFTWARE.
       If any other value is specified then requests will use GET.
     </st:attribute>
   </st:documentation>
-  <f:prepareDatabinding />
   <j:choose>
-    <j:when test="${h.useHidingPasswordFields()}">
-      <j:set var="value" value="${h.getPasswordValue(attrs.value ?: instance[attrs.field])}"/>
+    <j:when test="${displayOnlyMode}">
       <j:choose>
-        <j:when test="${ value != null }">
-          <st:adjunct includes="lib.form.password.password"/>
-          <div class="hidden-password">
-            <m:input xmlns:m="jelly:hudson.util.jelly.MorphTagLibrary"
-                     class="complex-password-field hidden-password-field setting-input ${attrs.checkUrl!=null?'validated ':''}${attrs.clazz}"
-                     name="${attrs.name ?: '_.'+attrs.field}"
-                     value="${value}"
-                     type="hidden"
-                     checkMethod="post"
-                     ATTRIBUTES="${attrs}" EXCEPT="field clazz value" />
-            <div class="hidden-password-placeholder">
-              <div class="hidden-password-legend">
-                <svg width="20px" height="25px" viewBox="0 0 25 32" version="1.1" xmlns="http://www.w3.org/2000/svg">
-                  <!--
-                      Based on Material Design.
+        <j:when test="${value}"><span class="jenkins-readonly">****</span></j:when>
+        <j:otherwise>
+          <span class="jenkins-not-applicable">N/A</span>
+        </j:otherwise>
+      </j:choose>
+    </j:when>
+    <j:otherwise>
+      <f:prepareDatabinding/>
+      <j:choose>
+        <j:when test="${h.useHidingPasswordFields()}">
+          <j:set var="value" value="${h.getPasswordValue(attrs.value ?: instance[attrs.field])}"/>
+          <j:choose>
+            <j:when test="${ value != null }">
+              <st:adjunct includes="lib.form.password.password"/>
+              <div class="hidden-password">
+                <m:input xmlns:m="jelly:hudson.util.jelly.MorphTagLibrary"
+                         class="complex-password-field hidden-password-field setting-input ${attrs.checkUrl!=null?'validated ':''}${attrs.clazz}"
+                         name="${attrs.name ?: '_.'+attrs.field}"
+                         value="${value}"
+                         type="hidden"
+                         checkMethod="post"
+                         ATTRIBUTES="${attrs}" EXCEPT="field clazz value"/>
+                <div class="hidden-password-placeholder">
+                  <div class="hidden-password-legend">
+                    <svg width="20px" height="25px" viewBox="0 0 25 32" version="1.1"
+                         xmlns="http://www.w3.org/2000/svg">
```

Comment:
Really unnecessary whitespace change, cannot even be hidden through `?w=1` on GitHub.